<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectDialog</class>
 <widget class="QDialog" name="NewProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="RussianFederation"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Название проекта:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="le_name"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Расположение файла проекта:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hl_1">
     <item>
      <widget class="QLineEdit" name="le_file"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_file">
       <property name="text">
        <string>Обзор...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Онтология шаблонов задач:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hl_2">
     <item>
      <widget class="QLineEdit" name="le_template">
       <property name="text">
        <string>/home/anton/Learning/Курсовая работа - зачёт/Текст квазибакалаврской работы/ontologies/LTemplates.owl</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_template">
       <property name="text">
        <string>Обзор...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Онтология предметной области:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hl_3">
     <item>
      <widget class="QLineEdit" name="le_domain">
       <property name="text">
        <string>/home/anton/Learning/Курсовая работа - зачёт/Текст квазибакалаврской работы/ontologies/LEntities.owl</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_domain">
       <property name="text">
        <string>Обзор...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="minimumSize">
        <size>
         <width>82</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>ОК</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="minimumSize">
        <size>
         <width>82</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
